Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -393,31 +393,17 @@ public void SendToAll<T>(T msg, int channelId = Channel.Reliable)
             NetworkConnection.Send(connections, msg, channelId);
         }
 
-        private readonly List<NetworkIdentity> DirtyObjectsTmp = new List<NetworkIdentity>();
-
         // The user should never need to pump the update loop manually
         internal void Update()
         {
             if (!Active)
                 return;
 
-            DirtyObjectsTmp.Clear();
-
             foreach (NetworkIdentity identity in DirtyObjects)
             {
                 if (identity != null)
-                {
-                    identity.ServerUpdate();
-
-                    if (identity.StillDirty())
-                        DirtyObjectsTmp.Add(identity);
-                }                
+                    identity.ServerUpdate();           
```

Comment:
Put this function back.

This has nothing to do with the change.

This was an optimization we had that prevents us from looping all the objects looking for dirty ones,  instead we only serialize the ones that are dirty.

So for us,  this is a regression

Reply:
fixed